packages/react-server-examples/bike-share/components/network-card.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {logging, Link} from 'react-server';
+
+const logger = logging.getLogger(__LOGGER__);
+
+const NetworkCard = ({network}) => {
+	logger.info(`rendering card for network ${JSON.stringify(network)}`);
+	return (
+		<div><Link path={`/network?network=${network.id}`} frameback reuseFrame>{network.name}</Link> in {network.location.city}</div>
```

Comment:
[jsx-boolean-value](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md) errors when I provide a truthy value; iiuc, `={true}` is no different than providing attribute alone.